tests/end_to_end/end_to_end.cpp

```diff
@@ -2768,7 +2771,7 @@ TEST_CASE("load_native_program_invalid3", "[end-to-end]")
 }
 TEST_CASE("load_native_program_invalid4", "[end-to-end]")
 {
-    _test_load_invalid_program("empty_um.dll", EBPF_EXECUTION_NATIVE, -EINVAL);
+    _test_load_invalid_program("empty_um.dll", EBPF_EXECUTION_NATIVE, 0);
```

Comment:
nit: Since `empty_um.dll` is now a valid program, we should not call `_test_load_invalid_program()`. Also since this is new code path where there are 0 programs and >0 maps in native module, I think the test should validate that the expected map is indeed created once the module is loaded.

Reply:
Updated